prow/tide/tide_test.go

```diff
@@ -1042,24 +1031,32 @@ func TestTakeAction(t *testing.T) {
 		} else if act != tc.action {
 			t.Errorf("Wrong action. Got %v, wanted %v.", act, tc.action)
 		}
-		if tc.triggered != len(fkc.createdJobs) {
-			t.Errorf("Wrong number of jobs triggered. Got %d, expected %d.", len(fkc.createdJobs), tc.triggered)
+
+		numCreated := 0
+		var batchJobs []*prowapi.ProwJob
+		for _, action := range fakeProwJobClient.Actions() {
+			switch action := action.(type) {
+			case clienttesting.CreateActionImpl:
+				numCreated++
+				if prowJob, ok := action.Object.(*prowapi.ProwJob); ok && prowJob.Spec.Type == prowapi.BatchJob {
```

Comment:
would also switch this to if !ok || pJ.S.T != pa.BJ { continue }

Reply:
we only do something in the positive case so that seems not most appropriate here

Reply:
Right, golang has a preference for identifying the negative case and exiting (and minimizing indentation of the positive code path).

Reply:
right but we would have one line indented in both cases

Reply:
and the `continue` case would be more LOC too